pkgs/by-name/ie/ietf-cli/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  rsync,
+  nix-update-script,
+}:
+python3.pkgs.buildPythonApplication rec {
+  name = "ietf-cli";
+  version = "1.27";
+  format = "other";
```

Comment:
`format = "other"` is deprecated — use `pyproject = false;` instead

Reply:
thanks; changed now